test/asm-parser-subclass-integration-tests.ts

```diff
@@ -50,17 +50,7 @@ function processAsmWithParser<T extends AsmParser>(
 }
 
 describe('AsmParser subclass compatibility', () => {
-    describe('labelFindFor method behavior', () => {
-        it('should use VcAsmParser labelFindFor override to find specific VC labels', () => {
-            const asmLines = ['_start:', 'mov eax, OFFSET _data', 'call _function', 'jmp _start'];
-            const usedLabels = initializeParserAndFindLabels(VcAsmParser, [], asmLines);
-
-            // VC-specific label detection should find these labels
-            expect(usedLabels.has('_data')).toBe(true);
-            expect(usedLabels.has('_function')).toBe(true);
-            expect(usedLabels.has('_start')).toBe(true);
-        });
```

Comment:
This test is duplicated in https://github.com/compiler-explorer/compiler-explorer/blob/f289b79f1d3699a89df1a0a98d5b186441ea2883/test/vc-asm-parser-tests.ts#L121-L133